app/components/UI/Ramp/buy/hooks/useFiatCurrencies.ts

```diff
@@ -0,0 +1,100 @@
+import { useRampSDK } from '../../common/sdk';
+import useSDKMethod from '../../common/hooks/useSDKMethod';
+import { useEffect, useMemo } from 'react';
```

Comment:
nit. Same as other hook
```suggestion
import { useEffect, useMemo } from 'react';
import useSDKMethod from '../../common/hooks/useSDKMethod';
import { useRampSDK } from '../../common/sdk';
```